test/FunctionalTests/Microsoft.OData.Core.Tests/IntegrationTests/Writer/JsonLight/ODataJsonLightWriterIntegrationTests.cs

```diff
@@ -748,7 +790,8 @@ private static string WriteJsonLightEntry(bool isRequest, Uri serviceDocumentUri
         private static string WriteJsonLightEntry(bool isRequest, Uri serviceDocumentUri, bool specifySet,
             ODataResource odataEntry, IEdmNavigationSource entitySet, IEdmStructuredType resourceType,
             ODataUri odataUri, Action<ODataWriter> writeAction = null, bool isResourceSet = false,
-            EdmModel model = null)
+            EdmModel model = null,
+            bool ValidatePropertyNames = true)

```

Comment:
maybe it's better to accept "ValidationKinds" enum?

Reply:
Good suggestion, but this is internal test code; we can change easily change it later if/when we add tests that handle other ValidationKinds.

---
In reply to: [154829788](https://github.com/OData/odata.net/pull/1010#discussion_r154829788) [](ancestors = 154829788)